packages/plugin-user/src/user.js

```diff
@@ -164,6 +164,31 @@ const User = SparkPlugin.extend({
   },
 
   /**
+   * Extracts a user object for a user specified by userId(UUID) and organization specified by orgId(UUID)
+   * @param {Object} options
+   * @param {string} options.userId
+   * @param {string} options.orgId
+   * @private
+   * @returns {Promise<Object>}
+   */
+  getUserForOrg(options) {
```

Comment:
Thanks @ianwremmel, I was thinking of the same. Unfortunately there is no userCache available today, and I do know that there is a lot of discussion around having another service which can do user lookup cross-org, since the user in orgA cannot lookup orgB (security issue as well). So for now, I will park this PR until that is resolved.